app/src/main/java/com/appunite/loudius/network/model/error/DefaultErrorResponse.kt

```diff
@@ -17,6 +17,6 @@
 package com.appunite.loudius.network.model.error
 
 data class DefaultErrorResponse(
-    val message: String,
```

Comment:
info: I fixed this code, because we had crashes in the app when API returned an error that didn't have those information provided.